duckduckgo/duckduckgo-privacy-extension
# Facebook Click to play

Reviewer: Dave

## Description:
Adds click to play functionality for Facebook. This replaces visible facebook elements with an element that a user can click to enable if they wish to view the content. After this change, no facebook requests will be sent until the user interacts with a facebook element.

Note that in this PR is a background HTML page for "learn more". This is likely to be removed before final merging, and instead be placed on our public help pages (still working with copy team).

## Steps to test this PR:
There is a test page that loads all the facebook elements (though login doesn't work on the test page, due to an invalid application ID from facebook) at http://privacy-test-pages.glitch.me/privacy-protections/click-to-load/ (note that metric counts are not accurate when using FireFox) 

In the network tab of dev tools, all facebook requests should be blocked until a user interacts with an element, at which point the element should load normally (may be slightly visually modified from it's original version). Login with facebook should work transparently on custom login buttons.

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
